packages/zero-client/src/client/context.ts

```diff
@@ -123,6 +135,14 @@ export class ZeroContext implements QueryDelegate {
         duration,
       );
     }
+
+    // Add a histogram for the materialization duration.
+    let histogram = this.#materializeHistograms.get(hash);
+    if (histogram === undefined) {
+      histogram = new LogarithmicHistogram();
+      this.#materializeHistograms.set(hash, histogram);
+    }
```

Comment:
where do the histograms get cleaned up?

Reply:
They are in memory and are very small. That is one of the benefits of histograms